docs/guides/understanding_the_mining_process.rst

```diff
@@ -380,4 +380,4 @@ zero value transfer transaction.
   ... )
 
   >>> chain.mine_block(mix_hash=mix_hash, nonce=nonce)
-  <ByzantiumBlock(#Block #1-0x41f6..2913)>
```

Comment:
Why did this change? Something about the gas limit being off by one?

Reply:
@carver yep